settings.gradle

```diff
@@ -0,0 +1,16 @@
+pluginManagement {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
```

Comment:
do we need this one? is this from the template?

Reply:
yeah I think I've seen it in the other PRs too